frontend/src/index.tsx

```diff
@@ -3,16 +3,15 @@ import ReactDOM from 'react-dom';
 import App from './App';
 
 import { createStore } from 'redux';
-import { Provider} from 'react-redux';
+import { Provider } from 'react-redux';
 
 // Redux State
-import reducer from "./store/reduxState";
+import reducer from './store/reduxState';
 const elixrStore = createStore(reducer);
 
 ReactDOM.render(
     <Provider store={elixrStore}>
         <App />
     </Provider>,
-    document.getElementById('root')
+    document.getElementById('root'),
```

Comment:
This is another common code style that I've used/seen... trailing commas.

It basically avoids annoying merge conflicts and makes changing less lines of code when adding to the end of the list.